src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/TabView.shared.cs

```diff
@@ -794,7 +794,8 @@ void UpdateSelectedIndex(int position, bool hasCurrentItem = false)
 					}
```

Comment:
Just notices some lines of code above can be simplified)
```csharp
if (index == position)
    TabItems[position].IsSelected = true;
else
    TabItems[index].IsSelected = false;
```
to
```csharp
TabItems[index].IsSelected = index == position;
```

Reply:
Do you mind doing it in another PR?

Reply:
sure)